config/sync/user.role.site_reviewer.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 789e9ccb-b42b-44e2-9dc4-dea61e0c4ac4
+langcode: en
+status: true
+dependencies: {  }
+id: site_reviewer
+label: 'Site Reviewer'
+weight: 4
+is_admin: null
+permissions:
+  - 'access toolbar'
+  - 'view any unpublished content'
```

Comment:
I don't think they need access to the toolbar do they? They won't have any buttons to click.

Reply:
I will remove that!

Reply:
@pookmish I think this is ready for a re-review. 

Since I don't need to test the toolbar now, the test is just stubbed out... testing nothing. I couldn't think of a good page to test for testing "unpublished." Does this test happen against the initial state? (In which case we could test one of the unpublished bits of default content?)